svelte/bootstrap/src/components/select/Select.svelte

```diff
@@ -82,43 +65,13 @@
 				</Slot>
 			</div>
 		{/each}
-		<input
-			id={$id$}
-			aria-label={$ariaLabel$}
-			type="text"
-			class="au-select-input flex-grow-1 border-0"
-			value={$filterText$}
-			aria-autocomplete="list"
-			autoCorrect="off"
-			autoCapitalize="none"
-			autoComplete="off"
-			on:input={onInput}
-			on:keydown={onInputKeydown}
-		/>
+		<input class="flex-grow-1 border-0" value={$filterText$} use:inputAttributesDirective />
```

Comment:
As `inputAttributesDirective` currently manages a class and here we also handle static classes, you would need to do the mergeDirectives with a new `createClassDirective`... You can reference what was done in Rating.svelte.

Reply:
Never mind, David will start working on the preprocessor soon, you can forget about this comment.